website/layouts/shortcodes/include.html

```diff
@@ -0,0 +1,4 @@
+{{ $file := .Get 0 }}
+{{ $fileContent := (printf "%s%s" .Page.File.Dir $file) | readFile }}
+{{ $renderedContent := $fileContent | replaceRE "^---[\\s\\S]+?---" "" | markdownify | safeHTML }}
```

Comment:
See earlier note on dropping frontmatter from "tutorial building block" files and then use the normal readfile function

Reply:
I have an issue with including another shortcode, even the description said it should be supported (https://www.docsy.dev/docs/adding-content/shortcodes/#include-external-files:~:text=Hugo%20v0.101.0%2B%20is%20required%20for%20imported%20files%20containing%20shortcodes%20to%20be%20rendered%20correctly.) 
I keep this shortcode for now.